### PR TITLE
Add tooltips

### PR DESCRIPTION
**Pull request by IagoLast**
#43 Tooltips
---

### CODE REVIEW

**Comment by piensaenpixel** [comment]
<3

File: packages/styles/src/tooltip/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Tooltips are small, interactive, textual hints for mainly graphical elements.
+
+Notice we only provide the `styles` and `css classes` required to display a tooltip you need to control the placement
```
Comment (VictorVelarde) [inline packages/styles/src/tooltip/README.md]: required. To display a tooltip

File: packages/styles/src/tooltip/tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  &::after {
+    content: "";
+    position: absolute;
+    border-width: 5px;
+    border-style: solid;
+    border-radius: 2px;
+    border-color: var(--as-color-type-01, #{$color-type-01});
+  }
+
+  &--bot::after {
```
Comment (VictorVelarde) [inline packages/styles/src/tooltip/tooltip.scss]: maybe we could use `bottom`instead of `bot`?

Comment (IagoLast) [reply packages/styles/src/tooltip/tooltip.scss]: we use bot all along the project

Comment (VictorVelarde) [reply packages/styles/src/tooltip/tooltip.scss]: so bot-it-is 🤖

**Pull request merged**